source/common/http/header_map_impl.h

```diff
@@ -480,7 +485,8 @@ class RequestHeaderMapImpl final : public TypedHeaderMapImpl<RequestHeaderMap>,
   };
 
   using HeaderHandles = ConstSingleton<HeaderHandleValues>;
-
+  std::string forwarding_path_;
```

Comment:
Should the intended use for these two be documented?

Reply:
Makes sense.